crates/types/src/invocation.rs

```diff
@@ -28,6 +28,7 @@ use std::time::Duration;
 
 // Re-exporting opentelemetry [`TraceId`] to avoid having to import opentelemetry in all crates.
 use crate::journal_v2::{CompletionId, GetInvocationOutputResult, Signal};
+use crate::GenerationalNodeId;
```

Comment:
Let's keep the newline between ours and external types:

```suggestion
use crate::GenerationalNodeId;

```